test/usbmux/usbmux-specs.js

```diff
@@ -0,0 +1,54 @@
+import fs from 'fs';
+import path from 'path';
+import net from 'net';
+import Usbmux from '../../lib/usbmux/usbmux';
```

Comment:
It would be better to pull this from `index.js` (i.e., `import { Usbmux } from '../..'`. That would give confirmation as well that the package itself can be imported.